src/storage/src/table/cell_based_table.rs

```diff
@@ -170,16 +185,34 @@ impl<S: StateStore> CellBasedTable<S> {
 
     pub async fn get_row_by_scan(&self, pk: &Row, epoch: u64) -> StorageResult<Option<Row>> {
         // get row by state_store scan
+        let vnode = self
```

Comment:
Why `CellBasedTable` need to compute vnode? I think this should be provided by executors creating CellBasedTable?

Reply:
It's just that cell based table provides such interface:
```
async fn batch_write_rows_inner<const WITH_VALUE_META: bool>
```
which I think indicates whether to compute value meta in cell based table. 🤔

Reply:
Value meta needs to be computed when write of course. But for reads, isn't it true that all executors and their state table objects already has value meta assigned to them? For both point get and scan, we should use vnode provided by executors to do filters, instead of compute it.

Reply:
So we should use the same set of vnodes to do pruning, regardless of type of the read operation (point-get or range-scan). cc. @fuyufjh

Reply:
For reads performed on a single vnode (e.g. we know the dist key beforehand), I think computing vnode on the fly makes sense. In other cases, I think we should just use the vnodes of the executor, which should be initialized on `CellBaseTable` initialization.